ksqldb-tools/src/test/java/io/confluent/ksql/tools/migrations/commands/ApplyMigrationCommandTest.java

```diff
@@ -276,12 +276,19 @@ public void shouldResetVariablesBetweenMigrations() throws Exception {
   @Test
   public void shouldApplyArgumentVariablesEveryMigration() throws Exception {
     // Given:
-    final Map<String, Object> variables = ImmutableMap.of("name", "tame");
-    command = PARSER.parse("-a", "-d", "name=tame");
-    createMigrationFile(1, NAME, migrationsDir, "INSERT INTO FOO VALUES ('${name}');");
-    createMigrationFile(2, NAME, migrationsDir, "INSERT INTO FOO VALUES ('${name}2');");
+    command = PARSER.parse("-a", "-d", "name=tame", "-d", "dame=blame");
+    createMigrationFile(1, NAME, migrationsDir, "INSERT INTO FOO VALUES ('${name}'); INSERT INTO FOO VALUES ('${dame}');");
+    createMigrationFile(2, NAME, migrationsDir, "DEFINE name='flame'; INSERT INTO FOO VALUES ('${name}');");
```

Comment:
Let's have a separate unit test for this piece that verifies the order of precedence between DEFINE statements within migrations files and the command options. (The unit test name does not clarify that this is being tested here, so it's unclear to the reader that this piece of the test is important. Unit tests are meant to be small and quick to run so it doesn't hurt to have more of them, as long as each tests a very targeted piece of functionality.)